src/web/tags.rs

```diff
@@ -32,12 +32,20 @@ impl Serialize for Tags {
     {
         let mut seq = serializer.serialize_map(Some(self.tags.len()))?;
         for tag in self.tags.clone() {
-            seq.serialize_entry(&tag.0, &tag.1)?;
+            if &tag.1 != "" {
```

Comment:
nit: to be consistent 
```suggestion
            if !tag.1.is_empty() {
```